Gopkg.toml

```diff
@@ -71,6 +71,12 @@ ignored = [
   name = "cloud.google.com/go"
   revision = "094187a3a68a589c8217a46b90af0efae38542b9"
 
+# https://github.com/xwb1989/sqlparser/issues/31
+# https://github.com/xwb1989/sqlparser/issues/32
+[[constraint]]
+  name = "github.com/xwb1989/sqlparser"
+  source = "https://github.com/dt/sqlparser"
```

Comment:
i'm real opposed to this name aliasing because it may be confusing to many. why not just depend on your package or else make one in the cockroach org?

Reply:
what do you mean by name aliasing?

this is depending on my remote, since the refs with the parser patches aren't available upstream yet. I can mirror it to the org as well, though I thought we usually only did that once we determined it would be long-lived.

Reply:
i mean why not import github.com/dt/sqlparser directly instead of the other one?

Reply:
my mirror just has a couple patches on top of upstream -- I didn't rewrite all the internal import paths, so it is just an alternative source for `xwb1989/sqlparser` -- it is not importable as `dt/sqlparser`.

Reply:
ah i didn't know about the import paths. ok this is fine then. thanks for explaining.